project.clj

```diff
@@ -62,7 +62,7 @@
                                   [puppetlabs/trapperkeeper ~tk-version :classifier "test"]
                                   [org.clojure/tools.namespace "0.2.4"]
                                   [org.clojure/java.jmx "0.2.0"]
-                                  [spyscope "0.1.4"]
+                                  [spyscope "0.1.5" :exclusions [clj-time]]
```

Comment:
OK I didn't realize you were saying that it had jumped up all the way to 0.7.0.  I think we do want to stay on 0.5.1 for the rest of the project... so, either explicitly add that and put the exclusion into spyscope like you mentioned, or, leave us on the old spyscope.  Don't care which one.